tokio-trace-fmt/src/span.rs

```diff
@@ -472,14 +472,14 @@ impl Slot {
 impl Slab {
     #[inline]
     fn write_slot(&self, idx: usize) -> Option<RwLockWriteGuard<Slot>> {
-        self.slab.get(idx).map(|slot| slot.write())
+        self.slab.get(idx).map(RwLock::write)
     }
 
     #[inline]
-    fn read_slot<'a>(&'a self, idx: usize) -> Option<RwLockReadGuard<'a, Slot>> {
```

Comment:
is this a semantic difference?

Reply:
Based on my understanding of lifetime ellision rules, no.

I remember from before though you mentioned `'_` not being supported in some versions, so I need to double check that this compiles in 1.26 locally.

Reply:
I can say right now that this won't compile on 1.26. I'd be fine with adding the clippy ignore attribute for that lint for now?